zephyr/esp32/CMakeLists.txt

```diff
@@ -368,32 +368,34 @@ if(CONFIG_SOC_SERIES_ESP32)
       ../../components/esp_phy/src/phy_common.c
       )
 
-    zephyr_sources_ifdef(
-      CONFIG_WIFI_BUILD_ONLY_MODE
-      ../port/wifi/wifi_stubs.c
-      ../port/phy/phy_stubs.c
-      )
-
-    zephyr_link_libraries_ifndef(
-      CONFIG_WIFI_BUILD_ONLY_MODE
-      net80211
-      core
-      pp
-      phy
-      rtc
-      ## esp-idf wifi libs refer gcc libs symbols, so linked in libgcc
-      gcc
-        -L${CMAKE_CURRENT_SOURCE_DIR}/../blobs/lib/esp32
-    )
-
+    if (NOT(CONFIG_WIFI_BUILD_ONLY_MODE OR CONFIG_BT_BUILD_ONLY_MODE))
```

Comment:
should it be AND instead of OR?
if only one is in build mode, the lib should be included as well

Reply:
Or perhaps use only one Kconfig for both cases, such as `CONFIG_RF_BUILD_ONLY_MODE`

Reply:
Or even more generic, for any feature that may depend on blobs, that that seems like the ultimate purpose that this is being introduced. E.g. something like `CONFIG_BUILD_ONLY_NO_BLOBS`.

Reply:
@jhedberg please read my previous comment at https://github.com/zephyrproject-rtos/zephyr/pull/80785#issuecomment-2453265212